tests/fixtures/encoder_decoder/nlvr_semantic_parser/ungrouped_experiment.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
It's fine to have other names in here for other configurations, as long as there is one that's just `experiment.json`, and that's the one you want to test with a `Predictor`.